include/sys/vdev_impl.h

```diff
@@ -265,15 +265,28 @@ struct vdev {
 #define	VDEV_PHYS_SIZE		(112 << 10)
 #define	VDEV_UBERBLOCK_RING	(128 << 10)
 
+/*
+ * MMP blocks occupy slots in the uberblock ring.
+ *
+ * VDEV_UBERBLOCK_COUNT()	slots uberblocks are stored in
+ * MMP_BLOCKS_PER_LABEL		slots used for MMP blocks
+ *
+ * Non-MMP aware ZFS uses the entire uberblock ring for uberblocks, so
+ * on import the entire ring must be checked for valid uberblocks.
+ */
+#define	MMP_BLOCKS_PER_LABEL	1
```

Comment:
I need to think this through.  If this were set to 0, bad things would probably happen, so I should fix that.